packages/@react-spectrum/datepicker/docs/DateRangePicker.mdx

```diff
@@ -287,6 +287,19 @@ This example only accepts dates after today.
 <DateRangePicker label="Date range" isDisabled />
 ```
 
+The `isDateDisabled` callback prop makes it possible to disable custom dates from the `RangeCalendar`. It is being called for each date and if it returns `true`, the date cell is disabled.
```

Comment:
Perhaps the wording can be changed to something like this? Same wording suggestion applies for the rest of the docs pages
```suggestion
The `isDateDisabled` callback prop allows you to disable specific dates within the calendar. Each date in the calendar is passed to the callback and is disabled if the callback returns `true`.
```
Still iffy on my own second sentence haha

Reply:
```suggestion
The `isDateDisabled` callback prop acts like a filter function. It is called for each visible date and if it returns `true`, the date cell is disabled.
```
maybe this is better?